test/jit_utils.py

```diff
@@ -271,51 +280,65 @@ def checkScriptRaisesRegex(self, script, inputs, exception, regex,
             ge = torch.jit.script(script, optimize)
             ge(*inputs)
 
+    def get_frame_vars(self, frames_up):
```

Comment:
Why is this necessary to add?

Reply:
To `exec` the function you need to pass the scope so it can lookup values